feature-libs/my-account/src/my-account.module.ts

```diff
@@ -1 +1,5 @@
+import { NgModule } from '@angular/core';
+import { CostCenterModule } from './company/cost-center/cost-center.module';
+
+@NgModule({ imports: [CostCenterModule] })
```

Comment:
I was expecting a CompanyModule sitting between cost center an myaccount. I also imagine that we don't import cost center here, as it's optional, and only used for b2b. 
If it's all nicely lazily loaded, it might be fine, but it's still adding overhead.

Reply:
Good point for the CompanyModule, it's better to create it in this PR.
It will be lazy loaded when Stan jumps in. But we can do this change in the mean time.